frontend/src/utils/projectPage.js

```diff
@@ -1,8 +1,60 @@
 import yaml from 'js-yaml'
 import { updateFile, uploadFile } from '@utils/giteaApi'
+import { flatten } from 'lodash'
 
 const processorUrl = process.env.KITSPACE_PROCESSOR_URL
 
+/**
+ * Given an array of kitspace repos return the an array of projects.
+ * Each multi project is treated as a standalone project.
+ * @param {object[]} repos
+ * @returns {Promise<[]object>}
+ */
+export const getFlatProjects = async repos => {
+  /**
+   * @param {string} fullname
+   * @returns {Promise<[boolean, object?]>} The first item is whether the project is multi, the second is the multi projects in kitspace.yaml.
+   */
+  const isMultiProject = async fullname => {
```

Comment:
The name suggest this is a boolean and not a function returning multi. Would suggest `getMultiInfo` that just returns the `multi` or `undefined`, not an array with redundant info.

Reply:
Resolved in  315842e.